public/Leo Jayachandran/Glossarizer/libretextsGlossarizer.js

```diff
@@ -303,7 +316,7 @@ class LibreTextsGlossarizer {
 
                 for (let i = 0; i < this.glossary.length; i++) {
                     if (this.options.exactMatch) {
-                        if (this.glossary[i].term.toLowerCase() == this.clean(term).toLowerCase()) {
```

Comment:
Originally this escaped punctuation so that the word can be used in regex, but since the glossary terms contain punctuation, the glossarizer previously would find a match but not be able to figure out which term matched, because the punctuation would be escaped by the clean function. Now the glossarizer is able to figure out which term matches even when there is punctuation like brackets